reference/docs-conceptual/community/community-support.md

```diff
@@ -0,0 +1,33 @@
+---
+title: PowerShell community resources
+description: List of resources created for and by PowerShell users
+ms.date: 03/04/2020
+---
+# PowerShell community resources
+
+The PowerShell Community is a vibrant and active group of users. This article can help you get
+connected with other member of the community.
+
+## Getting support from the community
+
+The PowerShell community can file issues, bugs, or feature requests in our
+[GitHub](https://github.com/powershell/powershell/issues) repository. If you questions, you may find
+help from other members of the community in one of these public forums:
+
+- [User Groups](https://aka.ms/psusergroup)
+- [PowerShell Tech Community](https://techcommunity.microsoft.com/t5/PowerShell/ct-p/WindowsPowerShell)
+- [DSC Community](https://dsccommunity.org/)
+- [PowerShell.org](https://powershell.org/)
+- [StackOverFlow](https://stackoverflow.com/questions/tagged/powershell)
+- [r/PowerShell subreddit](https://www.reddit.com/r/PowerShell/)
+- [PowerShell Slack](http://poshcode.org/slack)
+- [PowerShell Discord](http://poshcode.org/discord)
```

Comment:
These should be https, but I see that pshcode.org has a cert problem. In the interim, might be best to use an aka.ms link to keep it as https, then switch back to poshcode.org once the cert issue is resolved.

Reply:
fixed